doc/content/guides/connecting-gateways/thethingskickstartergateway/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "The Things Kickstarter Gateway"
+description: ""
+weight: 3
+menu:
+  main:
+    weight: 3
+---
+
+The Things Kickstarter Gateway is a LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.thethingsnetwork.org/docs/gateways/gateway/). This page guides you to connect it to The Things Stack.
+
+
+## Prerequisites
+
+1. User account on The Things Stack with rights to create Gateways and API Keys.
+2. The Things Gateway running the latest firmware (a minimum of `v1.0.7` is necessary).
+
+## Registration
+
+Login to The Things Stack via the CLI/console.
+
+Create a new gateway with the desired `Gateway ID` and choose the correct frequency plan for your device. The EUI field can be left blank. For details on using the CLI/Console, refer to the [getting started]({{< ref "/guides/getting-started" >}}) section.
```

Comment:
1. Make Gateway ID bold instead of code
2. IIRC there were some extra restrictions for the ID, because the gateway's web ui does some validation on it. So either we need to fix that, or we need to tell users what those restrictions are.